package.json

```diff
@@ -104,5 +104,8 @@
     "sinon": "1.17.2",
     "sinon-chai": "2.8.0",
     "webpack-livereload-plugin": "^0.11.0"
+  },
+  "optionalDependencies": {
```

Comment:
this is a dupe of @billyvg's work, i can back it out, but it looks like he hadn't landed it yet and is OOO for a bit